src/raft.c

```diff
@@ -347,6 +348,17 @@ void RaftExecuteCommandArray(RedisRaftCtx *rr,
         size_t cmdlen;
         const char *cmd = RedisModule_StringPtrLen(c->argv[0], &cmdlen);
 
+        if (strncasecmp("watch", cmd, cmdlen) == 0) {
```

Comment:
`strncasecmp("watch", "w", 1) == 0` will enter this if check. So, I suggest adding len check first:

```c
if (cmdlen == 5 && strncasecmp("watch", cmd, cmdlen) == 0) {
```

Reply:
ok, makes sense.

Reply:
done